src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -228,7 +228,13 @@ private FilePath GetNuGetPath()
                 arguments.Append("-Prerelease");
             }
 
-            arguments.Append("-ExcludeVersion -NonInteractive -NoCache");
+            // NoCache
+            if (definition.Parameters.ContainsKey("nocache"))
```

Comment:
Would be nice if you could add unit tests for this (to make sure it's only included when it's set).